blocks/graph.py

```diff
@@ -205,8 +205,9 @@ def replace(self, replacements):
         for node in apply_nodes_sorted:
```

Comment:
If you also find this identical to `for variable in self.variables`, then maybe we should we switch to it, because it's a way easier.